draft-irtf-cfrg-hpke.md

```diff
@@ -1207,6 +1207,27 @@ across all labels in this document.
 The `0xFFFF` AEAD ID is reserved for applications which only use the Export
 interface; see {{hpke-export}} for more details.
 
+# API Considerations {#api-considerations}
+
+The high-level HPKE APIs specified in this document are all fallible.
```

Comment:
Nit: Are Seal() and Export() necessarily fallible? I guess Seal() is vaguely fallible at counter overflow, or if you pick EXPORT_ONLY. Export() seems infallible from the specification's perspective. (Of course, it may be fallible in the implementation if, say, allocation fails. But that's an implementation-specific detail.)

Reply:
They're both fallible if the input sizes [are simply enormous](https://cfrg.github.io/draft-irtf-cfrg-hpke/draft-irtf-cfrg-hpke.html#name-input-length-restrictions), so I just bundled them all together here. (But that may not be too different from a `malloc` failure 🤷.) I dunno. This seems not wrong, so I'm inclined to keep it.

Reply:
Hehehe, fair enough, although I don't think you typically have that much address space. :-)